Source/ASTableView.mm

```diff
@@ -51,6 +51,26 @@
     return __val; \
   }
 
+#define UITABLEVIEW_RESPONDS_TO_SELECTOR() \
+  ^{ \
+    static BOOL superResponds; \
+    static dispatch_once_t onceToken; \
+    dispatch_once(&onceToken, ^{ \
+      superResponds = [UITableView instancesRespondToSelector:_cmd]; \
+    }); \
+    return superResponds; \
+  }()
```

Comment:
Nice! Rather than specifying this as a block, let's just make it a scope:

```objc
#define UITABLEVIEW_RESPONDS_TO_SELECTOR() \
  ({ \
    static BOOL superResponds; \
    static dispatch_once_t onceToken; \
    dispatch_once(&onceToken, ^{ \
      superResponds = [UITableView instancesRespondToSelector:_cmd]; \
    }); \
    superResponds; \
  })

```